polaris-react/src/components/Text/Text.tsx

```diff
@@ -60,6 +62,8 @@ export interface TextProps {
   variant?: Variant;
   /** Visually hide the text */
   visuallyHidden?: boolean;
+  /** Add a line-through to the text */
+  textDecorationLine?: TextDecorationLine;
```

Comment:
Should we make this a boolean prop since we're only supporting `line-through`? 

```suggestion
 strikethrough?: boolean;
```

Reply:
It's a good point, I left it open in case we wanted to allow other values later on... debating on this :)